chain/chain/src/test_utils.rs

```diff
@@ -682,7 +682,8 @@ impl RuntimeAdapter for KeyValueRuntime {
 
         Ok(ApplyTransactionResult {
             trie_changes: WrappedTrieChanges::new(
-                self.trie.clone(),
+                self.get_tries(),
```

Comment:
we need exact shard at this point, not all of them
why not to use `get_trie_for_shard` instead?

Reply:
because later we pass all of them to `StoreUpdate`